src/services/motion/types.ts

```diff
@@ -72,13 +72,13 @@ export const encodeMotionConfiguration = (
   const result = new pb.MotionConfiguration();
 
   result.setObstacleDetectorsList(
-    obj.obstacleDetectorsList.map((x: ObstacleDetector) => {
+    obj.obstacleDetectorsList?.map((od: ObstacleDetector) => {
```

Comment:
I needed to add this, otherwise the caller gets an error if `obstacleDetectorsList` (which is supposed to be optional) is undefined.